tests/plugins/test_deepspeed_plugin.py

```diff
@@ -483,6 +505,23 @@ def configure_optimizers(self):
         }]
 
 
+class ManualModelParallelClassificationModel(ModelParallelClassificationModel):
+
+    def automatic_optimization(self) -> bool:
```

Comment:
something is odd here
property missing or not?

so when Lightning calls model.automatic_optimization it gets a function as a value which evaluates to True?

Reply:
good eye, this should be set in the `__init__`

Reply:
ah yep missing the @property here, thanks @awaelchli